.travis.yml

```diff
@@ -3,12 +3,13 @@ cache:
   directories:
     - WcaOnRails/vendor/bundle
 rvm:
-  - 2.3.1
+  - 2.4.0
```

Comment:
I [just learned](https://github.com/larspetrus/Birdflu/pull/2) that you can have a `.ruby_version` file in your project that both rbenv and rvm will obey.

I think we should have that in this project, since it would help people run the right ruby version.

I also suspect/hope that the rvm directive here could then be removed.